src/components/structures/MatrixChat.tsx

```diff
@@ -1460,6 +1461,36 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
                 showNotificationsToast(false);
             }
 
+            if (!localStorage.getItem("mx_seen_feature_thread_experimental")) {
+                setTimeout(() => {
+                    if (SettingsStore.getValue("feature_thread") && SdkConfig.get()['showLabsSettings']) {
+                        Modal.createDialog(InfoDialog, {
+                            title: _t("Threads are no longer experimental! 🎉"),
+                            description: <>
+                                <p>
+                                    { _t("We’ve recently introduced key stability "
+                                    + "improvements for Threads, which also means "
+                                    + "phasing out support for experimental Threads.") }
+                                </p>
+                                <p>
+                                    { _t("All thread events created during the "
+                                    + "experimental period will now be rendered in "
+                                    + "the room timeline and displayed as replies. "
+                                    + "This is a one-off transition. Threads are now "
```

Comment:
While translating on Weblate I noticed that this line did not really match the one on the image inside the PR description. According to the image, this line should be `"This is one-off transition, and threads are now "`. Is this fine?

Reply:
The screenshot was out of date, we iterated on the text a bit

Reply:
OK I see, thanks for checking!